### ISSUE TITLE
HamlLint::RakeTask#config ineffective?

### ISSUE THREAD

**Issue by tisba** [brigade/haml-lint#277]
According to README and source, I should be able to do this in my `Rakefile` to 

```ruby
require 'haml_lint/rake_task'

HamlLint::RakeTask.new do |t|
  t.config = "config/haml-lint.yml"
end
```

But if I try to use the task, the configuration in `config/haml-lint.yml` is simply ignored. Having the same file in the projects root directory as `.haml-lint.yml` works as expected though.

Am I overlooking something?
